lib/model/rwfolder.go

```diff
@@ -191,7 +191,7 @@ func (f *rwFolder) Serve() {
 				l.Debugln(f, "skip (initial)")
 				f.pullTimer.Reset(f.sleep)
 			} else {
-				prevVer, prevIgnoreHash = f.onExpiredPullTimerfunc(prevVer, prevIgnoreHash)
+				prevVer, prevIgnoreHash = f.updatePreviousVersionAndPreviousIgnoreHashOnExpiredPullTimer(prevVer, prevIgnoreHash)
 			}
```

Comment:
Sry for that long name `updatePreviousVersionAndPreviousIgnoreHashOnExpiredPullTimer`. This gives me a hint there is more than one responsibility in this method. Just stopped here for a while.